content/en/monitors/notifications.md

```diff
@@ -142,15 +142,15 @@ For example, if your tag is `dot.key.test:five` and your monitor is grouped by `
 #### Log facet variables
 
 Log monitors can use facets as variables if the monitor is grouped by the facets.
-For example, if your log monitor is grouped by the facet `@facet`, the variable is:
+For example, if your log monitor is grouped by the `facet`, the variable is:
 
 ```text
-{{@facet.name}}
```

Comment:
Are you sure the `.name` should be deleted here? on line 153, you left it in.

Reply:
You are very correct! Good catch. I must've fat-fingered that one out of there. Thanks!